src/__tests__/PostPreview.test.tsx

```diff
@@ -0,0 +1,10 @@
+import {screen, render} from "@testing-library/react";
+import PostPreview from "../components/PostPreview";
+
+describe('PostPreview test', ()=>{
```

Comment:
please provide a more descriptive message on what are you testing f.e. component renders with no errors